src/assets/base.css

```diff
@@ -78,14 +78,24 @@ body {
   color: var(--color-background);
   background: none;
   line-height: 1.5;
-}
 
-.input-style:focus {
-  outline: none;
-}
+  &::placeholder {
+    color: var(--color-background);
+  }
+
+  &:focus {
+    outline: none;
+  }
+
+  &:-webkit-autofill,
+  &:-webkit-autofill:hover,
+  &:-webkit-autofill:focus,
+  &:-webkit-autofill:active {
+    -webkit-text-fill-color: #ffffff;
+    -webkit-box-shadow: 0 0 0 30px var(--color-primary-1) inset !important;
```

Comment:
No !important in css please.

If important is needed, there is issue with precedence, and css structure need to be review and changed